### ISSUE TITLE
Not compatible with Monk's Enhanced Journal

### ISSUE THREAD

**Issue by Shayulghul** [Xbozon/storyteller#35]
Just want to throw this out here that it doesn't work with Monk's Enhanced Journal when it did before the update.

**Comment by rtschaefer** [comment]
Yes, same here. All of my stories disappeared and the button to create stories is gone. I does look like some update with Monk's Enhanced Journal is conflicting.  I really hope the issue can be resolved, and that I did not lose all of my stories.

**Comment by Xbozon** [comment]
Stories are definitely not gone, most likely if you turn off Monks-journal, they will appear. So far I do not understand how to make these two mods work together.

**Comment by mfozz** [comment]
I love Storyteller, but I also have a need for Monk's Enhanced Journal, so I decided just to roll back Storyteller to v3.4. Not a big deal. I actually like having my stories on a different tab.

**Comment by rtschaefer** [comment]
Can confirm in my case that turning off Monk's Enhanced Journal does not fix the issue.   I will try rolling back Storyteller to v3.4 and hopefully that will remedy the issue.

**Comment by Xbozon** [comment]
Right now, stories are created just like Journals, check out the updated repository home page. Could this be what you mean?

**Comment by rtschaefer** [comment]
Well yes, but stories were reduced to simple journals.  I reverted back to v3.4 and, awesome news, all of my stories with artwork and everything are back as they were before!  Huge sigh of relief for me.  Hours upon hours of Photoshop work went into the art.   I love Storyteller, it is an amazing module, and I hope there are more enhancements forthcoming.

**Comment by rtschaefer** [comment]
I noticed an update. I am nervous to try it.  Does this fix the issue? When it broke the Story button was gone, and all of my stories were just text, not the cool book.

**Comment by rtschaefer** [comment]
Nevermind... the update was just appearing because I went back to v3.4 (LOL).  I hope to see a new version that will allow me to continue using the most recent versions of both modules without issues.  Thanks again... I love Storyteller!

**Comment by Xbozon** [comment]
@rtschaefer

Unfortunately, the automatic restore cannot change the default style. I restored the text and image, please change the style manually using these instructions.

 https://github.com/Xbozon/storyteller/issues/33#issuecomment-1186501096

**Comment by rtschaefer** [comment]
Good day! 

I was curious to see if there is a planned upgrade to fix compatibility with Monk's Enhanced Journal. I noticed that an Alpha version 4.x is out....

**Comment by Xbozon** [comment]
Try it now, maybe after the update the problem is no longer there.

**Comment by ChobitsCrazy** [comment]
I came to report the same issue, love this mod and would like to use it with Monk's. Here is what I found on the current release.
1. If you create a story page without Monk's then you can view it okay as expected.
2. If you enable Monk's any Story pages already created kind of work, they show inside of the journal in a strange way but still functional esp if you disable the players from using Monk's in the settings.
3. While both are enabled creating a Story page simply does not work.

Thanks for the great mod!

**Comment by ScottUlmer** [comment]
> I came to report the same issue, love this mod and would like to use it with Monk's. Here is what I found on the current release.
> 
>     1. If you create a story page without Monk's then you can view it okay as expected.
> 
>     2. If you enable Monk's any Story pages already created kind of work, they show inside of the journal in a strange way but still functional esp if you disable the players from using Monk's in the settings.
> 
>     3. While both are enabled creating a Story page simply does not work.
> 
> 
> Thanks for the great mod!

Exact thing thing I am finding. One other note for PLAYERS, it works perfectly, it is not "embedded" in the enhanced journal.
